backend/src/main/models/AdminAccount.ts

```diff
@@ -0,0 +1,21 @@
+import {
```

Comment:
Did prettier put the imports in different lines?

Reply:
Yeah